SU-SWS/stanford_profile
# D8CORE-1280: fix for intro text styles

# Not READY FOR REVIEW - advice needed


# Summary
- TL;DR - Fixes the styles for intro text

# Needed By (Date)
- Before sprint end

# Urgency
- Would be needed for any site using these styles

# Steps to Test

1. Check out this branch
1. Clear Cache
1. In the editor add "Intro text" to text
1. Save.
1. Verify that the Intro text is large and bold.

![Screen Shot 2020-02-19 at 5 45 57 PM](https://user-images.githubusercontent.com/284440/74892906-cd433400-533f-11ea-9deb-c06b166e318a.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1280
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
Please see my [request for assistance](https://stanfordits.atlassian.net/browse/D8CORE-1279?focusedCommentId=90556&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-90556)

- Anyone who should be notified? (`@mention` them here)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
